docs/sources/panels/configure-visualizations/index.md

```diff
@@ -0,0 +1,112 @@
+---
+aliases:
+  - /docs/grafana/latest/panels/working-with-panels/apply-color-to-series/
+  - /docs/grafana/latest/panels/working-with-panels/configure-legend/
+title: Configure visualizations
+menuTitle: Configure visualizations
+weight: 50
+keywords:
+  - color
+  - visualization
+  - legend
+---
+
+# Configure visualizations
+
+To configure a visualization, you can apply color to series and fields, and configure a legend.
```

Comment:
Don't think this page makes any sense (Sorry missed it in the doc). I

Sounds like a generic topic but is really about series color, and legend (two super random visualization-specific settings out of many). 

think part of this content can move into this page
https://grafana.com/docs/grafana/next/panels/standard-field-definitions/ 

that page details the standard options, and Color scheme is part of that, but that page does not document the color option, 

The legend option is part of Time series, Barchart and Pie chart visualizations. So think we should move that to a separate file and embedd that content into each of those visualization docs.

Then we can remove this "Configure visualizations" page

Reply:
ok, awesome feedback, thank you!